### PR TITLE
adding alert to check out map

### PR DESCRIPTION
**Pull request by evantarrh**
screenshot 2015-04-03 14 24 13

Also adding a script block so that only capacity.html has the expand js from @RaymondXu.

### CODE REVIEW

File: density/static/capacity.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	// This function expands the about container to
+	// the reveal additional information available on
+	// a non-mobile view.
+	$("#expand-collapse").click(function() {
+		if(! $("#learn-more").hasClass("open")) {
+			$("#learn-more").html("<strong>COLLAPSE</strong>");
+			$("#learn-more").addClass("open");
+			$("#expandable").addClass("expanded");
+		}
+		else {
```
Comment (natebrennand) [inline density/static/capacity.js]: do you guys have a style guide for JS yet?

[airbnb](https://github.com/airbnb/javascript) has one that a lot of people use. Generally they suggest: `} else {`

**Comment by evantarrh** [comment]
good call––also, tabs vs spaces is sloppy here, i'll fix that

**Comment by natebrennand** [comment]
There are tabs vs spaces issues in other JS files too I think. Wanna clean them all up in this PR?

**Comment by evantarrh** [comment]
yeah, that's a good idea; I think the only other JS in the project is in the map page, though. I'll clean it up and make an actual map.js file instead of just putting it inline in map.html

**Comment by natebrennand** [comment]
ok, if you're gonna move it out to a different file it might be more appropriate to make it a separate PR

**Comment by evantarrh** [comment]
okay, sounds good

**Comment by natebrennand** [comment]
:+1:

**Comment by RaymondXu** [comment]
:+1:

**Pull request merged**